### PR TITLE
WAZO-2172: call record start

### PR DESCRIPTION
**Pull request by bloom1**
Depends-On: https://github.com/wazo-platform/wazo-calld-client/pull/36

This PR is based on the "record stop" PR: https://github.com/wazo-platform/wazo-calld/pull/141
You should also check the wazo-auth-keys PR: https://github.com/wazo-platform/wazo-auth-keys/pull/34
You should also check the wazo-asterisk-config PR: https://github.com/wazo-platform/wazo-asterisk-config/pull/32

### CODE REVIEW

**Comment by bloom1** [comment]
recheck

File: CHANGELOG.md
Comment (fblackburn1) [inline CHANGELOG.md:7]:
```suggestion
  * `record_state`
```

File: wazo_calld/plugin_helpers/ami.py
Comment (fblackburn1) [inline wazo_calld/plugin_helpers/ami.py:43]: Why we cannot use ari to get variable like other endpoints?

**Review by fblackburn1 — CHANGES_REQUESTED** [review]
Can you split test_calls to test_call_records file? This file is definitively too long

**Review by fblackburn1 — CHANGES_REQUESTED** [review]
Triggering multiple start will start simultaneous MixMonitor and you will need to trigger the same amount of stop to really stop de recording
We should verify that call is not currently recorded before sending another MixMonitor command

**Review by fblackburn1 — APPROVED** [review]

**Review by fblackburn1 — APPROVED** [review]

**Comment by fblackburn1** [comment]
@bloom1 FYI I have no objection if you set `auto-merge` label after quick fix that dismissed my approval :)

**Comment by bloom1** [comment]
> @bloom1 FYI I have no objection if you set `auto-merge` label after quick fix that dismissed my approval slightly_smiling_face

Hehe alright :smile:

**Pull request merged**
